src/identity.ts

```diff
@@ -1,3 +1,11 @@
+/**
+ * A function that always returns the param passed to it
+ * @signature
+ *    R.identity(data)
+ * @example
+ *    R.identity('foo') // => 'foo'
+ * @category Function
+ */
 export function identity<T>(value: T) {
```

Comment:
documented this as well because it's exported but doesn't show up in the docs :)